plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreServerExtension.java

```diff
@@ -104,6 +104,11 @@ public interface PostgreServerExtension {
     String getProceduresSystemTable();
     String getProceduresOidColumn();
 
+    /**
+     * Returns true if a database contains prosp column in the pg_proc system table. This column knows about routine type.
+     */
+    boolean supportsProSPColumn();
```

Comment:
Lets not add yet another functin to ServerType. We can determine column presence in loadInfo (lookup in resultset metadata)